ares/component/processor/v30mz/instructions-misc.cpp

```diff
@@ -21,7 +21,7 @@ auto V30MZ::instructionLock() -> void {
 }
 
 auto V30MZ::instructionWait() -> void {
-  wait(1);
+  wait(1 + 9);
```

Comment:
Any reason why 1 +9 rather than 10?

Reply:
It's not strictly necessary, but the V30MZ datasheet lists the timing for this opcode as the rather cryptic `Number of 9 + 9 × POLLB pin samplings`, which I *assume* means "1 + 1 * 9", so I wanted to reflect that here.

Reply:
But I changed it, as I don't strongly feel about it either way.